package.json

```diff
@@ -122,7 +122,9 @@
       "!src/stories/*",
       "!src/components/AddAccommodationForm.js",
       "!src/views/Dashboard/addAccommodationPage.js",
-      "!src/helpers/*"
+      "!src/views/Message.js",
+      "!src/helpers/*",
+      "!src/redux/actions/accommodationAction.js"
```

Comment:
What the use of these lines

Reply:
@AJAkimana These lines are there to tell jest for not covering these files